### ISSUE TITLE
Creating or reloading a dial removes custom title of bookmark

### ISSUE THREAD

**Issue by AdelSulaiman** [MatMoul/quickdial-webext#101]
I use over 900 bookmarks in Firefox, and I made it a habit of giving them custom short descriptive titles. But creating a dial or reloading a dial with Quick Dial swaps the custom title for the title of the website itself. Most of the times these titles are long and not usable.
Other than that a great add-on! Thank you for sharing this!

**Comment by MatMoul** [comment]
Actually, I've provided a checkbox in properties of each dials that can lock the title...
But with 900 bookmarks, it's not a possible way...
I still open this issue as a request can be treated in the futur...

**Comment by AdelSulaiman** [comment]
Yes, I have noticed the checkbox. Of course, you can only use the checkbox after creation of the dial, and by then the custom title has already been replaced.
Nevertheless, thank you for considering finding a solution in the future, I can ask no more.

**Comment by Neytrino-OnLine** [comment]
Similar problem, I use Private Bookmarks extension https://addons.mozilla.org/en-US/firefox/addon/webext-private-bookmarks/?src=search this expansion adds the folder with private bookmarks after unblocking, and deletes it when You lock it back. So all my private bookmarks - are creating every time i unlock it and QD changes their curent names to web-page provided name... :(
